src/dialogs/zha-device-info-dialog/dialog-zha-device-info.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  LitElement,
+  html,
+  css,
+  CSSResult,
+  TemplateResult,
+  customElement,
+  property,
+} from "lit-element";
+import "@polymer/paper-dialog-scrollable/paper-dialog-scrollable";
+import "../../components/dialog/ha-paper-dialog";
+import "../../panels/config/zha/zha-device-card";
+
+import { PolymerChangedEvent } from "../../polymer-types";
+import { haStyleDialog } from "../../resources/styles";
+import { HomeAssistant } from "../../types";
+import { ZHADeviceInfoDialogParams } from "./show-dialog-zha-device-info";
+import { ZHADevice, fetchDevice } from "../../data/zha";
+import { ZHADeviceRemovedEvent } from "../../panels/config/zha/types";
+
+@customElement("dialog-zha-device-info")
+class DialogZHADeviceInfo extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() private _params?: ZHADeviceInfoDialogParams;
+  @property() private _error?: string;
+  @property() private _device!: ZHADevice;
+
+  public async showDialog(params: ZHADeviceInfoDialogParams): Promise<void> {
```

Comment:
make sure to also reset error.

Reply:
You resolved this comment but I don't see `this._error = undefined` being added ?

Reply:
Oh, you do so in openedChanged. Sure that works too!

btw there is still a race condition in this code. The second time the dialog is opened, you set `this._params = params` and then `await fetchZHADevice`.  Since you do not reset `this._device = undefined`, your render method will actually already render the dialog but with the wrong device. 

You will need to set `this._device=undefined` to openedChanged too.